GitSavvy.sublime-settings

```diff
@@ -102,6 +102,22 @@
      */
     "close_panel_for": [],
 
+    /*
+        Whether git and githook output is logged live
+        If false, results are shown after execution.
+     */
+    "live_panel_output": true,
```

Comment:
Agree with @randy3k this should be false by default, at least for the time being.